Userland/Libraries/LibVideo/VP9/Utilities.h

```diff
@@ -1,22 +1,24 @@
 /*
  * Copyright (c) 2021, Hunter Salyer <thefalsehonesty@gmail.com>
+ * Copyright (c) 2022, Gregory Bertilson <zaggy1024@gmail.com>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #pragma once
 
+#include <AK/Error.h>
 #include <AK/Types.h>
 
 namespace Video::VP9 {
 
-#define SAFE_CALL(call)             \
-    do {                            \
-        if (!(call)) [[unlikely]] { \
-            dbgln("FAILED " #call); \
-            return false;           \
-        }                           \
-    } while (0)
+template<typename OutputType, typename InputType>
+ErrorOr<OutputType> verify_cast(InputType value)
+{
+    auto value_cast = static_cast<OutputType>(value);
+    VERIFY(value == static_cast<InputType>(value_cast));
+    return value_cast;
+}
```

Comment:
Can't we use something in AK/TypeCasts.h for this?

Reply:
Now that I think about it, this function was unnecessary in light of my later discovery of the Clang integer sanitizer, so I've removed this.